### PR TITLE
Added spelling notes and words

### PR DESCRIPTION
**Pull request by evildmp**

### CODE REVIEW

File: docs/how_to/custom_plugins.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ up-to-date at all times.
 It's like magic, but quicker.
 
 Unless you're lucky enough to discover that your needs can be met by the
-built-in plugins, or by the many available 3rd-party plugins, you'll have to
+built-in plugins, or by the many available-party plugins, you'll have to
```
Comment (vxsx) [inline docs/how_to/custom_plugins.rst]: should it not be `available third-party plugins` ?

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,7 +1,8 @@
 -f http://simple.crate.io/
 MarkupSafe==0.23
 Pygments==2.0.2
-Sphinx==1.3.1
-
-sphinx-autobuild==0.5.0
-sphinx_rtd_theme # always use latest version of the theme
+sphinx
```
Comment (vxsx) [inline docs/requirements.txt]: i think package name is `Sphinx`

**Comment by vxsx** [comment]
otherwise LGTM

File: docs/how_to/custom_plugins.rst (hunk 1 above)
Comment (evildmp) [inline docs/how_to/custom_plugins.rst]: aaargh

**Comment by vxsx** [comment]
:+1:

**Comment by FinalAngel** [comment]
LGTM

File: docs/requirements.txt (hunk 2 above)
Comment (evildmp) [inline docs/requirements.txt]: I didn't think names were case-sensitive.

Comment (vxsx) [inline docs/requirements.txt]: well, TIL then, seems to work for me locally on clean cache, so let's merge! :)

Comment (evildmp) [inline docs/requirements.txt]: BTW, ``pip install sphinx`` is actually what the official docs say, so I think it doesn't matter.

**Pull request merged**